### PR TITLE
support filtering attribute

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/ConfigFilterVisitor.cs [hunk 1]
```diff
@@ -29,21 +29,41 @@ protected override bool CanVisitApiCore(ISymbol symbol, bool wantProtectedMember
...
+            if (symbol == null)
+            {
+                throw new ArgumentNullException("symbol");
+            }
+
+            if (symbol.Kind == SymbolKind.Method && symbol.ContainingType != null)
+            {
+                symbol = symbol.ContainingType;
+            }
+
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/ConfigFilterVisitor.cs]: move to default filter visitor?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

**Comment by vwxyzh** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

**Pull request merged**